galaxy_mobile/lib/utils/switch_page_helper.dart

```diff
@@ -83,22 +83,17 @@ class SwitchPageHelper {
         .toList();
 
     // Update video slots.
-    // oldVideoFeeds.forEach((feed) => {
-    //       if (feed != null) {feed["videoSlot"] = -1}
-    //     });
-    oldVideoFeeds.isNotEmpty
-        ? oldVideoFeeds.forEach((feed) {
-            if (feed != null) {
-              feed["videoSlot"] = null;
-            }
-          })
-        : null;
-    newVideoFeeds.isNotEmpty
-        ? newVideoFeeds.forEach((feed) {
-            var index = newVideoFeeds.indexOf(feed);
-            if (feed != null && feed.isNotEmpty) feed["videoSlot"] = index;
-          })
-        : null;
+    oldVideoFeeds.forEach((feed) {
```

Comment:
@yanivel you removed the check for null, was it on purpose?

Reply:
Yes, we don't need to check if it's not empty since forEach will not do anything, so I simplified the code.